addons/shortcuts/statics.lua

```diff
@@ -198,9 +198,10 @@ end
 -- Iterate through resources and make validabils.
 function validabils_it(resource)
     for id,v in pairs(res[resource]) do
-        if (not v.monster_level and v.prefix) or (v.monster_level and v.monster_level ~= -1 and v.ja:sub(1,1) ~= '#' ) then
+        if (not v.monster_level and v.prefix) or (v.monster_level and v.monster_level ~= -1 and v.ja:sub(1,1) ~= '#' ) or (v.category == 'Usable') then
```

Comment:
This limits the items added to ones in the `Usable` category, which seem to be items like medicines, food, and clusters, but doesn't include usable gear, like `Warp Ring`. This seems fine to me, but highlighting this change in case someone else has a different opinion.

Reply:
This seems okay to me. If someone wants to revisit this decision, they can do it in another PR.

Reply:
I don't see a reason why it couldn't shouldn't check equipped items and make those usable via command.